skills/cypilot/scripts/cypilot/utils/ui.py

```
@@ -0,0 +1,269 @@
+        human_fn(data)
+        return
+
+    # Generic fallback
+    status = data.get("status", "")
+    message = data.get("message", "")
+    if status in ("PASS", "OK", "DRY_RUN"):
+        success(f"Done ({status})" + (f" — {message}" if message else ""))
+    elif status in ("FAIL", "ERROR"):
+        error(message or status)
+    elif status == "ABORTED":
+        warn(f"Aborted" + (f": {message}" if message else ""))
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)